jsonrpc/eth_endpoint.go

```diff
@@ -37,22 +34,16 @@ type ethStateStore interface {
 
 type ethBlockchainStore interface {
 	// Header returns the current header of the chain (genesis if empty)
-	Header() *types.Header
-
-	// GetHeaderByNumber returns the header by number
-	GetHeaderByNumber(block uint64) (*types.Header, bool)
+	Header() *Block
```

Comment:
Why does header function return block? =)

Reply:
Yes, the name should be changed. The idea is that for the `jsonrpc` module there is no difference between `Block` and `Header`. Thus, we should return only `jsonrpc.Block`. The name has to change too.